src/reducers/posts.js

```diff
@@ -67,7 +67,58 @@ const updatePostProps = (state, postId, props) => {
   }
 }
 
-export default function (state = {}, action) {
+const mockState = {
+  '123': {
+    id: '123',
+    name: 'The recently updated project',
+    description: '<p>Everything you want to know</p>',
+    created_at: '2017-01-11T00:27:30.522Z',
+    updated_at: '2017-01-11T02:52:03.036Z',
+    numComments: 1,
+    tag: 'ThisTagWillSoonBeGone',
+    type: 'project',
+    community_ids: [
+      '29'
+    ],
+    voter_ids: [],
+    follower_ids: [
+      '11204'
+    ],
+    user_id: '11204',
+    newActivity: {
```

Comment:
@levity - how do you feel about this structure in the reducer.
We need the post and the comment, and users for both. Right now it's all denormalized. I thought about adding clauses to the post, comment and user reducers to pull out the nested stuff, but couldn't decide what was better (and this way is definitely easier ;) )